Python/codegen.c

```diff
@@ -792,6 +792,17 @@ codegen_process_deferred_annotations(compiler *c, location loc)
         return SUCCESS;
     }
 
+    instr_sequence *old_instr_seq = INSTR_SEQUENCE(c);
+    instr_sequence *nested_instr_seq = NULL;
```

Comment:
Is there a reason why we can't reorder the code in this function so that we first emit the instructions for annotations and then for the code (so we don't need to do this injection)?